### ISSUE TITLE
Actual Excel Themes

### ISSUE THREAD

**Issue by myoder020** [ChronicStone/typed-xlsx#6]
Any plans on implementing the theming we get in Excel, like the actual table styles?
I tried applying a theme using the cellStyle but the theme index number isn't the same as what we get in Excel so I'm having to guess.

Also any way to enable filtering on a column?

**Comment by ChronicStone** [comment]
Hi @myoder020 regarding excel themes, I'm not sure, the library uses [xlsx-js-style](https://www.npmjs.com/package/xlsx-js-style), a fork of SheetJs supporting cell styling internally, so you should look into their docs / issues to see if that's possible.

Regarding column filtering, I'm not sure what you mean by that. Can you give me a bit more details, and maybe show what kind of API you would expect?

**Comment by myoder020** [comment]
My use case is just for basic reports.
They don't involve merged cells or sheets with multiple tables.

**Comment by ChronicStone** [comment]
@myoder020 Yeah but I need to make sure this does not break the library on more complex cases. The basic use case seem to work, I just need to fine tune a few things
<img width="1199" alt="image" src="https://github.com/ChronicStone/typed-xlsx/assets/55083156/88cbe81b-b2c6-4e39-9133-e134c1570983">

**Comment by ChronicStone** [comment]
That feature got a few caveats, I'll need to polish this, will try to ship it before the end of the week if I can find workaround for the edge cases

**Comment by myoder020** [comment]
Thanks for looking into it
